src/App.tsx

```diff
@@ -1,24 +1,203 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import {
+  getTodos,
+  getActiveTodos,
+  getCompletedTodos,
+  postTodos,
+  deleteTodos,
+} from './api/todos';
 
-const USER_ID = 0;
+import { Todo as TodoType } from './types/Todo';
+import { Todo } from './components/Todo';
+import { Filter } from './components/Filter';
+import { TempTodo } from './components/TempTodo';
+import { Error } from './components/Error/Error';
+import { FilterParams } from './components/Filter/FilterParams';
+
+const USER_ID = 9925;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<TodoType[]>([]);
+  const [filterParam, setFilterParam] = useState(FilterParams.All);
+  const [error, setError] = useState(false);
+  const [activeTodos, setActiveTodos] = useState(0);
+  const [completedTodos, setCompletedTodos] = useState(0);
+
+  const [inputValue, setInputValue] = useState('');
+  const [isGetError, setGetError] = useState(false);
+  const [isPostError, setPostError] = useState(false);
+  const [isDeleteError, setDeleteError] = useState(false);
+  const [isInputEmpty, setEmptyInputState] = useState(false);
+  const [isInputLocked, setLockInput] = useState(false);
+  const [tempTodo, setTempTodo] = useState<string | null>(null);
+  const [isClearAllCompleted, setClearAllCompleted] = useState(false);
+
+  const handleErrorState = (
+    setErrorFunc: (state: boolean) => void,
+    state = true,
+  ) => {
+    setError(state);
+    setErrorFunc(state);
+
+    if (state === false) {
```

Comment:
You can use logical operators instead
```suggestion
    if (!state) {
```

Reply:
Done